readme.rst

```diff
@@ -26,3 +26,16 @@ The following are use to generate the specification:
 
  - html:
     make html
+
+Status
+======
+
+As of February 2023 the spec is in the process of being finalised. Once this is
+complete and everyone is in agreement with the content, we will issue a 0.9
+release. We will then look to implement it in various target projects, including
```

Comment:
minor: I personally would have avoided these first 2 sentences as they're going to have to be updated again once the 0.9 release is out. But that's no big deal.